docs/application/dotnet/guides/applications/application_lifecycle.md

```diff
@@ -0,0 +1,196 @@
+        }
+```
+
+The following code snippet is responsible for initializing `Accelerometer` sensor. Function setups the event handler for data update and set the update interval for one second. After setup, the `SensorAccelerometer.Start()` to start measurements:
+
+```csharp
+        void InitSensors()
+        {
+            SensorAccelerometer = new Accelerometer();
+            SensorAccelerometer.Interval = 1000;
+            SensorAccelerometer.DataUpdated += OnAccelerometer;
+            SensorAccelerometer.Start();
+        }
+```
+
+UI setup is based on basic template. Only single `TextLabel` component is used to show the application UI and its running state:
+```csharp
+        void InitUI()
+        {
+            Window.Instance.KeyEvent += OnKeyEvent;
+
+            TextLabel test = new TextLabel("Lifecycle test...")
+            {
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                TextColor = Color.Blue,
+                PointSize = 12.0f,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                WidthResizePolicy = ResizePolicyType.FillToParent
+            };
+
+            Window.Instance.GetDefaultLayer().Add(test);
+        }
+```
+
+`OnTerminate()` method logs information about the application termination:
+
+```csharp
+        protected override void OnTerminate()
+        {
+            Tizen.Log.Info("LifecycleTest", "OnTerminate()");
+            base.OnTerminate();
+        }
+```
+
+`OnPause()` method is called when application is suspended, but not killed. Then, it goes into the `Paused` state. To stop reading data from the `Accelerometer`, `SensorAccelerometer.Stop()` method is used. Now, the application reduces resource usage. The main application window goes into the background now. UI events are not received, but the data event handler have to be stopped manually:
+
+```csharp
+        protected override void OnPause()
+        {
+            Tizen.Log.Info("LifecycleTest", "OnPause()");
+            base.OnPause();
+
+            if (SensorAccelerometer != null) SensorAccelerometer.Stop();
+        }
+```
+
+The `Tizen` operating system may resume `Lifecycle` application, if it is not terminated. Data updating is restored using the `SensorAccelerometer.Start()` method. The application main window should be visible now:
+
+```csharp
+        protected override void OnResume()
+        {
+            Tizen.Log.Info("LifecycleTest", "OnResume()");
+            base.OnResume();
+
+            if (SensorAccelerometer != null) SensorAccelerometer.Start();
+        }
+```
+
+The callback setup in the function `InitSensors()` is called for every second. It prints debug information about a measured data from `Accelerometer` sensor. In the emulator case, this values can be changed using `Control Panel` tool. For details, please check [Log Viewer](/application/dotnet/guides/sensors/device-sensors.md):
```

Comment:
Since the file **application_lifecycle.md** is currently missing in the files changed list in the PR, I checked the changes using the staging link. There is still issue with this hyperlink. The link is still broken.

Reply:
@sagarkedari: I noticed that application_lifecycle.md currently is in different place and it was modified by another PR's. That's why I removed old file from this PR. Invalid log viewer link should be fixed by another PR due this PR don't modify this file.